### ISSUE TITLE
Mark migration as already executed

### ISSUE THREAD

**Issue by Zarun1** [lecaillon/Evolve#190]
I'm working on a database project where we use Evolve to handle migrations. A team member carelessly made changes to the production database without using Evolve. After realizing the situation, we added migration scripts matching those changes to version control. But now we have a production database where those changes have already been made, without Evolve knowing about it. The changes added new table columns that now have data in them, so we can't simply reverse the changes and then make them again using Evolve.

Would it be possible to add a feature to Evolve that allows marking migration scripts as already executed, without actually executing them? I believe it would only need to add rows to the database version history table that show the migrations have been run.

This is a feature supported by Flyway Teams (skipExecutingMigrations option). They've implemented this as part of the migrate command.

**Comment by lecaillon** [comment]
First, did you manage to "fix" the Evolve metadata table ? @Zarun1 

Then, it is an interessant idea just need to think about the easiest way to implement it in Evolve.

**Comment by Zarun1** [comment]
I did. First I manually added rows to the metadata table so that it looked like I used Evolve to run the migrations. Then I used "evolve repair" command to fix the checksums. The end result worked well.

**Comment by lecaillon** [comment]
What about an option called `MarkNextMigrationApplied` used with the command migrate ? When run, Evolve would mark the next available migration successful and stop.

- Example with the Evolve CLI :

```
evolve migrate postgresql -c "Server=127.0.0.1;Database=..." -l "C:\db\migrations" --mark-next-migration-applied=True
```

**Comment by lecaillon** [comment]
Available since **Evolve 3.0.0-alpha6**
